jmp-cloud-service-impl/src/main/java/com/epam/learning/backendservices/rest/RESTApplication.java

```diff
@@ -0,0 +1,16 @@
+package com.epam.learning.backendservices.rest;
+
+import com.epam.learning.backendservices.rest.config.SpringConfig;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Import;
+
+@SpringBootApplication
+@Import({SpringConfig.class})
```

Comment:
You do not have to explicitly import configuration class. Spring framework will scan all packages under com.epam.learning.backendservices.rest. Optional: delete explicit import.

Reply:
Done